src/architect/account/account.utils.ts

```diff
@@ -70,6 +84,7 @@ export default class AccountUtils {
       }
     } else {
       inquirer.registerPrompt('autocomplete', require('inquirer-autocomplete-prompt'));
+      const can_run_local = await this.shouldListLocalAccounts();
```

Comment:
I'm not 100% sure if in this case we want to log a warning message to users. Example scenario here:
![image](https://user-images.githubusercontent.com/1406391/184677617-e3757426-34bc-43bf-919f-b4281ba4a8d2.png)

I think for the most part, if I don't have docker running, it's not that surprising to not have the local option and this warning message is just kind of superfluous. For a brand new user that maybe doesn't have docker running and would get confused, they probably don't have any remote accounts yet and would hit the hard-fail state above.

I'm leaning towards not logging anything if we're not hard-exiting, curious if anyone has thoughts